doc/testing-your-application.rst

```diff
@@ -3,45 +3,28 @@
+This chapter describes how to test your application against you reverse proxy
+instance.
 
-VarnishTestCase
----------------
-
-The FOSHttpCache library provides a base test case to help you write functional
-tests against a Varnish instance. This may be helpful to test the way your
-application sets caching headers and invalidates cached content.
+The FOSHttpCache library provides base test case classes to help you write 
+functional tests. This may be helpful to test the way your application sets 
+caching headers and invalidates cached content.
 
-By having your test classes extend ``VarnishTestCase``, you get:
+By having your test classes extend one of the testCase classes, you get:
 
-* independent tests: all previously cached content is removed by restarting
-  Varnish in the test setUp method
-* ``$this->varnish`` referring to an instance of this library’s Varnish client
-  that is configured to talk to your Varnish server
+* independent tests: all previously cached content is removed in the tests 
+  ``setUp`` method. The way this is done depends on which reverse proxy you use;
+* an instance of this library’s client that is configured to talk to your 
+  reverse proxy server. See reverse proxy specific sections for details;
 * convenience methods for executing HTTP requests to your application:
-  ``$this->getHttpClient()`` and ``$this->getResponse()``
+  ``$this->getHttpClient()`` and ``$this->getResponse()``;
 * custom assertions ``assertHit`` and ``assertMiss`` for validating a cache hit/miss.
 
-Configuration
--------------
-
-By default, the ``VarnishTestCase`` starts and stops a Varnish server for you.
-All you have to do, is to refer the class to your Varnish configuration (VCL)
-file. The recommended way to configure the test case is by setting constants
-in your `phpunit.xml`. Alternatively, you can override the getter methods:
-
-======================= ========================= ============ ==========================================
-Constant                Getter                    Default      Description
-======================= ========================= ============ ==========================================
-``VARNISH_FILE``        ``getConfigFile()``                    your Varnish configuration (VCL) file
-``VARNISH_BINARY``      ``getBinary()``           ``varnishd`` your Varnish binary
-``VARNISH_PORT``        ``getCachingProxyPort()`` ``6181``     port Varnish listens on
-``VARNISH_MGMT_PORT``   ``getVarnishMgmtPort()``  ``6182``     Varnish management port
-``WEB_SERVER_HOSTNAME`` ``getHostName()``                      hostname your application can be reached at
-======================= ========================= ============ ==========================================
+The recommended way to configure the test case is by setting constants
+in your `phpunit.xml`. Alternatively, you can override the getter methods.
 
 Setting Constants
-~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
this is wrong. It should have the same length than the title (Sphinx requires at least the length, but the Symfony standards are to use the same length)